packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -21,30 +22,40 @@ import {
   serializeDict,
   SolutionConfigError,
   ProjectSettingError,
+  environmentManager,
 } from "../..";
 import { LocalSettingsProvider } from "../../common/localSettingsProvider";
 import { Middleware, NextFunction } from "@feathersjs/hooks/lib";
 import fs from "fs-extra";
 import path from "path";
-import { readJson, checkFileExist } from "../../common/fileUtils";
+import { readJson } from "../../common/fileUtils";
 import { PluginNames } from "../../plugins/solution/fx-solution/constants";
 import { FxCore } from "..";
-import {
-  isMultiEnvEnabled,
-  isArmSupportEnabled,
-  isBicepEnvCheckerEnabled,
-} from "../../common/tools";
+import { isMultiEnvEnabled, isArmSupportEnabled, getStrings } from "../../common/tools";
 import { loadProjectSettings } from "./projectSettingsLoader";
 import { generateArmTemplate } from "../../plugins/solution/fx-solution/arm";
 import { loadSolutionContext } from "./envInfoLoader";
-import { ArmParameters, EnvConfigName, ResourcePlugins } from "../../common/constants";
+import { ArmParameters, ResourcePlugins } from "../../common/constants";
 import { getActivatedResourcePlugins } from "../../plugins/solution/fx-solution/ResourcePluginContainer";
-
-const MigrationMessage =
-  "In order to continue using the latest Teams Toolkit, we will update your project code to use the latest Teams Toolkit. We recommend to initialize your workspace with git for better tracking file changes.";
+import { LocalDebugConfigKeys } from "../../plugins/resource/localdebug/constants";
 
 const programmingLanguage = "programmingLanguage";
 const defaultFunctionName = "defaultFunctionName";
+const learnMoreText = "Learn More";
+const migrationSuccessMessage =
```

Comment:
also move this message to `resources/strings.json`?

Reply:
by the way, we need to mention we will create some new resources for bot / tab instead of reusing existing resources

Reply:
will add these message on wiki page